install/kubernetes/cilium/charts/hubble-tls/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+You have successfully installed Cilium {{ .Chart.Name }}
```

Comment:
minor nit: I would have put a period at the end here (and below) but it's not worth holding up the PR for this.